manifests/f/FreedomScientific/FreedomScientificJAWS2022/2022.2202.38.400/FreedomScientific.FreedomScientificJAWS2022.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FreedomScientific.FreedomScientificJAWS2022
+PackageVersion: 2022.2202.38.400
+PackageLocale: en-US
+Publisher: Freedom Scientific
+PackageName: Freedom Scientific JAWS 2022
+License: Copyright (c) Freedom Scientific. All rights reserved.

```

Comment:
A copyright is not a license; Please update this with proper license terms such as `Proprietary`, `MIT`, `GPL-2.0` or other terms as applicable